packages/eslint-plugin/src/rules/no-empty-function.ts

```diff
@@ -138,12 +140,25 @@ export default util.createRule<Options, MessageIds>({
       return false;
     }
 
+    function isAllowedEmptyOverrideMethod(
+      node: TSESTree.FunctionExpression,
+    ): boolean {
+      if (isAllowedOverrideMethods && isBodyEmpty(node)) {
+        return (
+          node.parent?.type === AST_NODE_TYPES.MethodDefinition &&
+          node.parent.override === true
+        );
+      }
+      return false;
```

Comment:
Nit: I think this can be simplified a bit?

```suggestion
      return (
        isAllowedOverrideMethods &&
        isBodyEmpty(node) &&
        node.parent?.type === AST_NODE_TYPES.MethodDefinition &&
        node.parent.override === true
      );
```